README.md

```diff
@@ -117,22 +109,21 @@ The XML snippet below shows the expected format for tags and debt remediation in
 </rules>
 ``` 
 
-#### Configuring NuGet feeds
+### Configuring NuGet feeds
 The SDK will look for NuGet.config files in the following locations:
-- in the directory containing *RoslynSonarQubeGenerator.exe*
-- %AppData%\NuGet (i.e. the standard pre-user location)
-- %ProgramData%\NuGet\Config\SonarQube (a custom machine-wide location
-- %ProgramData%\NuGet\Config (i.e. the standard machine-wide location)
+- in the directory containing `RoslynSonarQubeGenerator.exe`
+- `%AppData%\NuGet` - the standard pre-user location)
+- `%ProgramData%\NuGet\Config\SonarQube` - a custom machine-wide location
+- `%ProgramData%\NuGet\Config` - the standard machine-wide location
 
-If the analyzer you want to package is available in a private NuGet feed then you will need to create an appropriate NuGet.config file to point to the private feed. Alternatively you can use the `/customnugetrepo:file:///PathToRepo` 
-parameter. This will overwrite the above mentioned NuGet behaviour.
+If the analyzer you want to package is available in a private NuGet feed then you will need to create an appropriate `NuGet.config` file to point to the private feed. Alternatively you can use the `/customnugetrepo:file:///PathToRepo` parameter. This will overwrite the above mentioned NuGet behaviour.
 
-#### Generating a jar for an analyzer that is not available from a NuGet feed
+### Generating a jar for an analyzer that is not available from a NuGet feed
 If you want to create a jar for Roslyn analyzer that is not available from a NuGet feed (e.g. an analyzer you have created on your local machine) you can specify a package source that points at a local directory containing the *.nupkg* file created by the standard Roslyn templates. See the [NuGet docs](https://docs.nuget.org/create/hosting-your-own-nuget-feeds) for more information.
 
 By default, the [NuGet.config](./RoslynPluginGenerator/NuGet.config#L16) file shipped with the RoslynSonarQubeGenerator has a local package source configured that points to `C:\LocalNugetFeed`.
 
-#### NuGet packaging information
+### NuGet packaging information
```

Comment:
The link about the nuget properties documentation could be updated to: https://learn.microsoft.com/en-us/nuget/reference/nuspec#required-metadata-elements

> The NuGet package properties are documented [here](http://docs.nuget.org/Create/Nuspec-Reference)...

As well as the SQ plugin properties documentation link, which contains a typo (the `>` at the end).